jcstress-wrapper/src/main/kotlin/komem/litmus/JCStressRunner.kt

```diff
@@ -0,0 +1,47 @@
+package komem.litmus
+
+import java.nio.file.Path
+import kotlin.system.exitProcess
+
+/**
+ * Note that this 'runner' is severely different from all others.
+ */
+object JCStressRunner : LitmusRunner() {
```

Comment:
Why `object`?
Do you need it to be singleton?

Reply:
No, but it doesn't hold any state either. Changed it to a class, makes sense to change other runners as well

Reply:
Yeap, I think keeping them as classes is better.